fedn/setup.py

```diff
@@ -21,7 +21,7 @@
         "grpcio~=1.34.0",
         "numpy~=1.19.5",
         "protobuf",
-        "pymongo",
+        "pymongo>=4.0",
```

Comment:
Is this patch not backwards compatible? Can we not support older versions of pymongo?